pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
   'mmh3>=4.0.1',
   'orjson>=3.9.12',
   'httpx>=0.27.0',
+  'portalocker>=2.8.2',
```

Comment:
can this be dev dep? We are generally pretty conscious of what we allow to bloat our dep tree